src/platform_impl/linux/x11/window.rs

```diff
@@ -120,7 +120,16 @@ impl UnownedWindow {
         pl_attribs: PlatformSpecificWindowBuilderAttributes,
     ) -> Result<UnownedWindow, RootOsError> {
         let xconn = &event_loop.xconn;
-        let root = event_loop.root;
+        // root should be a type ffi::Window and it is finally c_ulong.
```

Comment:
Remove this comment entirely, there's no value in it. The WindowID thing is clear.

Reply:
fixed